Add documentation how to use sequel plugins (#19)
hanami/guides: content/models/database-configuration.md

```shell
$ HANAMI_ENV=test bundle exec hanami db prepare
```

## Sequel plugins

> Hanami models uses rom as a low level backend. It's mean that we can easy use any sequel plugins in your app.

Reviewer: ```suggestion
Hanami models use [ROM](https://rom-rb.org/) as a low-level backend. This means that you can easily use any [Sequel](https://github.com/jeremyevans/sequel) plugins in your app.
```

> For this you need to define a `gateway` block in your model configuration and add the extension by calling `extension` on `gateway.connection` and passing the name in:

Reviewer 2: ```suggestion
For this you need to define a `gateway` block in your model configuration, add the extension by calling `extension` on `gateway.connection` and pass the extension name in:
```
